# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/apz/util/ChromeProcessController.h
# Commit: da8c34fec36e
# Full Hash: da8c34fec36e933ccfe93e458ccdbe9328132897
# Author: Mike Conley <mconley@mozilla.com>
# Date: 2022-06-22 03:32:25
# Regressor Bug: 1773865
# File Overlap Count: 1
# Description:
#   Bug 1773865 - Dispatch an event on the window document when a pinch zoom gesture ends. r=botond,smaug,NeilDeakin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D149283
# ==============================================================================

diff -r 2510b654e8f2 -r da8c34fec36e gfx/layers/apz/util/ChromeProcessController.h
--- a/gfx/layers/apz/util/ChromeProcessController.h	Tue Jun 21 13:56:53 2022 +0000
+++ b/gfx/layers/apz/util/ChromeProcessController.h	Tue Jun 21 14:02:19 2022 +0000
@@ -75,6 +75,8 @@
   void NotifyAsyncAutoscrollRejected(
       const ScrollableLayerGuid::ViewID& aScrollId) override;
   void CancelAutoscroll(const ScrollableLayerGuid& aGuid) override;
+  void NotifyScaleGestureComplete(const ScrollableLayerGuid& aGuid,
+                                  float aScale) override;
 
   PresShell* GetTopLevelPresShell() const override { return GetPresShell(); }
 